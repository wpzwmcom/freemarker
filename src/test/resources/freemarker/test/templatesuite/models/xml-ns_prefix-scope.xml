<?xml version="1.0" encoding="UTF-8"?>
<root xmlns="http://freemarker.org/test/namespace-test"
    xmlns:foo="http://freemarker.org/test/foo"
    xmlns:bar="http://freemarker.org/test/bar">
  <e>e in NS namespace-test</e>
  <foo:e>e in NS foo</foo:e>
  <bar:e>e in NS bar</bar:e>
</root>