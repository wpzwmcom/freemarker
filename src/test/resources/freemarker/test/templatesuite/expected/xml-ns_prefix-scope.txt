//e: e in NS namespace-test, e in NS namespace-test
//n:e: e in NS foo, e in NS foo
//bar:e: e in NS bar, e in NS bar

Included:
//n:e: e in NS foo, e in NS foo
//n:e: e in NS foo, e in NS foo

Imported:
//n:e: e in NS bar, e in NS bar
//n:e: e in NS bar, e in NS bar
